rest-server/engines/src/main/scala/com/actionml/engines/ur/URDataset.scala

```diff
@@ -163,6 +153,44 @@ class URDataset(engineId: String, val store: Store) extends Dataset[UREvent](eng
     }
   }
 
+  override def inputMany(data: Seq[String]): Unit = {
+    val aliases = params.indicators.flatMap { ip =>
+      ip.aliases.getOrElse(Seq(ip.name))
+    }
+    val (events, items) = data.view.foldLeft[(List[UREvent], List[UREvent])]((List.empty, List.empty)) { case (acc@(el, il), jsonEvent) =>
```

Comment:
This is some pattern matching that I do not understand: `case (acc@(el, il)` what does it do? what ii it called?

@andreyshikov

Reply:
I'll answer in slack